Add Warning message on DNS Tokens (#607)
jellyfin/jellyfin.org: docs/general/networking/caddy.md

If you aren't familiar with Caddy yet, check out its [Getting Started](https://caddyserver.com/docs/getting-started) guide.

:::caution

> There are a some guides that have a Caddyfile which includes the `tls` section with a DNS Provider API Key, as shown the following example.

Reviewer: ```suggestion
There are a some guides that have a Caddyfile which includes a `tls` section with the DNS provider's API key as shown the following example.
```

> There are a some guides that have a Caddyfile which includes a `tls` section with the DNS provider's API key as shown the following example.

Reviewer: ```suggestion
There are a some guides that have a Caddyfile which includes a `tls` section with the DNS provider's API key as shown in the following example.
```

```Caddyfile
example.com {
    reverse_proxy 127.0.0.1:8096
    // highlight-start
    tls {
        dns <DNS Provider> <API Token>
    }
    // highlight-end
}
```

Please proceed with caution when using this option:

- This will **NOT** automatically update your DNS records if you have a dynamic IP.

> - This is **only** necessary if you **cannot** open ports 80 and 443

Reviewer: ```suggestion
- This is **only** necessary if you **cannot** open ports 80 and 443.
```

> - This is **NOT** required for automatic SSL to work in most cases.

Reviewer: ```suggestion
- This is **NOT** required for automatic TLS to work in most cases.
```

> - Misconfiguration can lead to **compromised domains and/or accounts**.

Reviewer 2: Is there a source or small explanation that you can back this up with? In what way can it lead to a compromise for example. Ignore this comment if this might be too much.

Author: I think it would be too long as this section is intended to be a short warning

Author: And also I think it should be very obvious simply by looking at what permissions are available and can be enabled for the API keys.

> - API Tokens should only be granted the least permissions for the application to function.

Reviewer: ```suggestion
- API keys should only be granted the least permissions required for the application to function.
```

> Please read [Let's Encrypt Documentation](https://letsencrypt.org/docs/challenge-types/) for more info.

Reviewer: ```suggestion
Please read the [Let's Encrypt documentation](https://letsencrypt.org/docs/challenge-types/) for more info.
```

If you already have access to the web interface, go to `Admin > Networking` and enter a path like `/jellyfin` in the Base URL field.
If not, you may instead go to `<Configuration Directory>/network.xml` and modify the value of `<BaseUrl>` according to your needs. For information on the directory location, please consult the [configuration documentation](/docs/general/administration/configuration#configuration-directory).
You might have to restart the Jellyfin server for this to take effect.

> Then simply give the `reverse_proxy` directive a path matcher. The path should be the same as the Base URL you entered into Jellyfin's settings. The following example is for a server that is under `example.com/jellyfin`.

Reviewer: ```suggestion
Then simply give the `reverse_proxy` directive a path matcher. The path should be the same as the `Base URL`  you entered into Jellyfin's settings. The following example is for a server that is under `example.com/jellyfin`.
```

> Then simply give the `reverse_proxy` directive a path matcher. The path should be the same as the `Base URL`  you entered into Jellyfin's settings. The following example is for a server that is under `example.com/jellyfin`.

Reviewer: ```suggestion
Then simply give the `reverse_proxy` directive a path matcher. The path should be the same as the `Base URL` you entered into Jellyfin's settings. The following example is for a server that is accessible at `example.com/jellyfin`.
```
